reverse-array.js

```diff
@@ -0,0 +1,25 @@
+// https://eloquentjavascript.net/04_data.html#i_6xTmjj4Rf5
+
+function reverseArray(array){
+  let newArray = [];
+  for (let item of array){
+    newArray.unshift(item);
+  }
+  return newArray;
+}
+
+function reverseArrayInPlace(array){
```

Comment:
This looks like the solution from the book.  Do you understand what's going on here?

Reply:
`Math.floor(array.length / 2)` is the middle of the array (in this case 3), so without touching 3, it swaps 1 (`array[i]' the first loop) with 5(`array[array.length - 1 - i]` the first loop) and then 2 with 4 the same way in the second loop.